scio-test/src/test/scala/com/spotify/scio/coders/KryoAtomicCoderTest.scala

```diff
@@ -38,87 +38,85 @@ import scala.collection.JavaConverters._
 import scala.reflect.ClassTag
 
 class KryoAtomicCoderTest extends PipelineSpec {
-
```

Comment:
Can we undo all the unnecessary style change here?

Reply:
yup